src/core/solver-base.cpp

```diff
@@ -115,33 +115,47 @@ double SolverAbstract::computeDynamicFeasibility() {
 
 double SolverAbstract::computeInequalityFeasibility() {
   tmp_feas_ = 0.;
+  Eigen::VectorXd g_adj;
```

Comment:
1. Declare this object as a private member of the SolverAbstract class.
2. Rename it to g_adj_